tests/testthat/test-run.R

```diff
@@ -36,6 +36,14 @@ test_that("run_one", {
   wipe_results()
 })
 
+test_that("get_params_summary returns a tibble",{
+  bm <- run_benchmark(placebo, cpu_count = 1, output_type = "message")
```

Comment:
It would be nice to also include a failure in this benchmark run. I'm _pretty sure_ you can `output_type = c("message", "abort")`

Reply:
Set this as `error_type` which I think works.